### ISSUE TITLE
Verification of static backend not working for cross domain setups

### ISSUE THREAD

**Issue by arkascha** [owncloud/shorty#115]
I thought I finally tackled all problems with static backend verification for cross domain setups. 
Apparently I was wrong. Obviously CORS headers won't get forwarded by apaches rewriting module (or whatever http server is used). Therefore the browser refuses to carry out the verification request. No idea why that did not show up in my tests. 
Unfortunately I cannot reenable to previous JSONP approach, since I became aware that it opened a huge security hole. So currently I am out of ideas, at least for normal user (administrators).

**Comment by arkascha** [comment]
OK, got a big step further: the validation of cross domain setup works! 
The requirements are documented in the CONFIGURATION doc file. 

However now there is another issue: 
since the cross domain setup requries the use of the proxy module/feature in request rewriting we now do not have a valid session when accessing the relay service via that proxy setup. Consequently a 403 error is returned for all requests, unfortunately no login is offered, so the service is currently useless for such setups.

**Comment by arkascha** [comment]
OK, seems I found a working solution and I am actually quite happy with it. 
The cross domain is obviously more complex, but I think this is ok, since it is a topic for advanced users anyway. Plus it is really slick now: when using the cross domain setup for static backends then the relay service is completely delivered via the second domain. That is transparent and reduces confusion for end users which clearly makes it preferable. 
The configuration doc file was updated accordingly. But this shows once more: Shorty does have to offfer that documentation inside the UI. Converting the doc files to markdown and creating some simple viewer is the way to go, I guess. 

Anyway, this entry can be closed for good :-)
